app/models/provider_enrichment.rb

```diff
@@ -34,7 +34,7 @@ class ProviderEnrichment < ApplicationRecord
 
   scope :latest_created_at, -> { order(created_at: :desc) }
   scope :latest_published_at, -> { order(last_published_at: :desc) }
-  scope :draft, -> { where(status: 'draft') }
+  scope :draft, -> { where(status: 'draft').or(ProviderEnrichment.rolled_over) }
```

Comment:
This looks like it should either be:

```suggestion
  scope :draft, -> { where(ProviderEnrichment.draft).or(ProviderEnrichment.rolled_over) }
```

or

```suggestion
  scope :draft, -> { where(status: [:draft, :rolled_over]) }
```

assuming either of these would actually work?

Reply:
that latter one seems to work :eyes: